docs/LocalesMenuButton.md

```diff
@@ -20,12 +20,7 @@ import { Typography } from '@mui/material';
 export const MyAppBar = (props) => (
     <AppBar {...props}>
         <Typography flex="1" variant="h6" id="react-admin-title"></Typography>
-        <LocalesMenuButton
-            languages={[
-                { locale: 'en', name: 'English' },
-                { locale: 'fr', name: 'Français' },
-            ]}
-        />
+        <LocalesMenuButton />
```

Comment:
Line 14: remove `, and list the locales available to end users`

Also, add something like

> **Tip:** the `<LocalesMenuButton>` will be added to the `<AppBar>` automatically if you have multiple locales declared in the `getLocales` method of your `i18nProvider`